nregress/ok/c99c11.bool.2

```diff
@@ -0,0 +1,2 @@
+In component `test':
```

Comment:
This is a sign of something not right (a valid input should not be producing this warning).